public/docs/ts/latest/guide/architecture.jade

```diff
@@ -358,52 +358,86 @@ figure
 
 .l-main-section
 :marked
-  ## Directives
+  ## Diretivas
 figure
   img(src="/resources/images/devguide/architecture/directive.png" alt="Parent child" style="float:left; width:150px; margin-left:-40px;margin-right:10px" )
 :marked
   Angular templates are *dynamic*. When Angular renders them, it transforms the DOM
   according to the instructions given by **directives**.
 
+  Templates no Angular são *dinâmicos*. Quando o Angular os renderiza, ele transforma o DOM
+  de acordo com as instruções passadas pelas as **diretivas**.
+
   A directive is a class with directive metadata. In !{_Lang} we apply the `@Directive` !{_decorator}
   to attach metadata to the class.
+
+  Uma diretiva é uma classe com metadados de uma diretiva. No !{_Lang} nós aplicamos o !{_decorator} `@Directive`
+  vincular os metadados para a classe.
 <br clear="all">
 :marked
   We already met one form of directive: the component. A component is a *directive-with-a-template*;
   a `@Component` !{_decorator} is actually a `@Directive` !{_decorator} extended with template-oriented features.
 
+  Nós já conhecemos um dos formatos da diretiva: componentes. Um componente é uma *diretiva-com-template*;
+  Um  !{_decorator} `@Component` é atualmente o !{_decorator} da `@Directive` estendida com as funcionalidades orientadas a template.
+
 .l-sub-section
   :marked
     While **a component is technically a directive**,
     components are so distinctive and central to Angular applications that we chose
     to separate components from directives in this architectural overview.
+
+    Enquanto **um componente é tecnicamente uma diretiva**,
+    componentes são tão característicos e fundamentais para aplicações angulares que escolhemos
+    separar componentes de directivas nesta visão geral da arquitetura.
+
 :marked
   Two *other* kinds of directives exist: _structural_ and _attribute_ directives.
 
+  Dois *outros* tipos de directivas são: _estruturais_ e _atributos_.
+
   They tend to appear within an element tag as attributes do,
   sometimes by name but more often as the target of an assignment or a binding.
 
+  Eles tendem a aparecer em um elemento como atributos HTML são,
+  às vezes pelo nome, mas frequentemente como destino de uma atribuição ou um binding.
+
   **Structural** directives alter layout by adding, removing, and replacing elements in DOM.
 
+  **Directivas estruturais** alteraram o layout adicionando, removendo e substituindo elementos no DOM.
+
   Our [example](#templates) template uses two built-in structural directives:
 
+  Nosso [template exemplo](#templates) usa duas diretivas contruídas usando esta estrutura:
```

Comment:
"Nosso template de exemplo usa duas diretivas estruturais **built-in**."

Acredito que a melhor traducao para "built-in" eh deixar em ingles. 😄